bool g_alarm    #global alarm
bool f_alarm    #front alarm
bool l_alarm    #left alarm
bool r_alarm    #right alarm
bool world_empty

float64 g_distance  #global distance
float64 f_distance  #front distance
float64 l_distance  #left distance
float64 r_distance  #right distance
float64 alarm_dir   #alarming direction
float64 wide_dir    #the widest direction from LIDAR
